optimum/habana/transformers/training_args.py

```diff
@@ -291,14 +291,6 @@ class GaudiTrainingArguments(TrainingArguments):
         metadata={"help": "Whether to use fp8 for training."},
     )
 
-    fp8_recipe_format: Optional[str] = field(
```

Comment:
@libinta please note we deleted this because we do not want to expose low level numeric selection to user as of now. In future we can revisit this.